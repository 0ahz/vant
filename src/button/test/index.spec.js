import { mount } from '../../../test';
import Button from '..';

test('loading-size prop', () => {
  const wrapper = mount(Button, {
    propsData: {
      loading: true,
      loadingSize: '10px',
    },
  });
  expect(wrapper.html()).toMatchSnapshot();
});
