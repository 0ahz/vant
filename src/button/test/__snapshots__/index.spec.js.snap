// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loading-size prop 1`] = `
<button type="button" class="van-button van-button--default van-button--normal van-button--loading">
  <div class="van-button__content">
    <div class="van-loading van-loading--circular van-button__loading"><span class="van-loading__spinner van-loading__spinner--circular" style="color: currentColor; width: 10px; height: 10px;"><svg class="van-loading__circular" viewBox="25 25 50 50"><circle cx="50" cy="50" r="20" fill="none"></circle></svg></span>
      <!---->
    </div>
    <!---->
    <!---->
  </div>
</button>
`;
